tutor/src/components/qa/index.cjsx

```diff
@@ -30,9 +30,10 @@ QADashboard = React.createClass
     )
 
   render: ->
+    return <h3 className="loading">Loading …</h3> if EcosystemsStore.isLoading()
+
     <div className="qa">
       <MatchForTutor {...@props} />
-      {<h3>Loading …</h3> if EcosystemsStore.isLoading()}
     </div>
```

Comment:
this is the bug fix, previously we were rendering the content even if the ecosystems were still loading